src/components/RightBoxSection.test.tsx

```diff
@@ -4,6 +4,7 @@ import { Ajax } from 'src/libs/ajax';
 import Events, { extractWorkspaceDetails } from 'src/libs/events';
 import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
 import { defaultAzureWorkspace } from 'src/testing/workspace-fixtures';
+import { WorkspaceRightBoxSection } from 'src/workspaces/dashboard/WorkspaceRightBoxSection';
```

Comment:
This test shouldn't be testing `WorkspaceRightBoxSection`. Move the "fires event" test case into `WorkspaceRightBoxSection.test.tsx`.